docs/README.md

```diff
@@ -76,3 +76,11 @@ agent_list = [
     agents.PlayerAgent(agent_control="wasd"), # W,A,S,D to move, E to lay bomb
 ]
 ```
+
+## Submitting an Agent.
+
+In order to submit an agent, you need to create an account at [pommerman.com](https://pommerman.com). This will link pommerman to your Github account. 
+
+Once you have created an account login and navigate to your profile, [Pommerman profile](https://pommerman.com/me). Here you will see a page where you can fill in a form that includes an agent name, git url (ssh is preferred), and the path to your agent's Docker file.
```

Comment:
"Once you have created an account login and navigate to your profile, [Pommerman profile]" --> "Once you have created an account, login and navigate to your profile page - [Pommerman profile]..."

Reply:
To submit an agent, fill in the form with your agent's name, an ssh git url, and the path to your agent's Docker file from the github repository's top level directory.